woocommerce/woocommerce
# [e2e] Update tests to pass on the Pressable env

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR attempts to fix tests not passing on the Pressable environment. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check the code. 
2. Make sure that CI is green and that tests are passing against wp-env.
3. Use [this](https://a8c-woo-test-reports.s3.amazonaws.com/reports/other/on_demand_e2eupdate-for-pressable-fcbd0d2/default-pressable-core-e2e-and-api/index.html) as a reference of test runs against Pressable. 
4. Run affected files against Pressable:

```
pnpm test:e2e:with-env default-pressable tests/e2e-pw/tests/admin-analytics/analytics-data.spec.js
```
```
pnpm test:e2e:with-env default-pressable tests/e2e-pw/tests/merchant/order-emails.spec.js
```
```
pnpm test:e2e:with-env default-pressable tests/e2e-pw/tests/merchant/products/block-editor/create-simple-product-block-editor.spec.js
```
```
pnpm test:e2e:with-env default-pressable tests/e2e-pw/tests/merchant/settings-email.spec.js
```
```
pnpm test:e2e:with-env default-pressable tests/e2e-pw/tests/shopper/mini-cart.spec.js
```





### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Update e2e tests so they're passing against Pressable environment

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
